apps/server/src/modules/files-storage/uc/files-storage.uc.ts

```diff
@@ -86,22 +86,27 @@ export class FilesStorageUC {
 
 	public async uploadFromUrl(userId: EntityId, params: FileRecordParams & FileUrlParams) {
 		await this.checkPermission(userId, params.parentType, params.parentId, PermissionContexts.create);
-		const response = await firstValueFrom(
-			this.httpService.get(params.url, {
-				headers: params.headers,
-				responseType: 'stream',
-			})
-		);
-
-		const fileDescription: IFile = {
-			name: decodeURI(params.fileName),
-			buffer: response.data as internal.Readable,
-			size: Number(response.headers['content-length']),
-			mimeType: response.headers['content-type'],
-		};
-		const result = await this.uploadFile(userId, params, fileDescription);
-
-		return result;
+		try {
+			const response = await firstValueFrom(
+				this.httpService.get<internal.Readable>(params.url, {
+					headers: params.headers,
+					responseType: 'stream',
+				})
+			);
+
+			const fileDescription: IFile = {
+				name: decodeURI(params.fileName),
+				buffer: response.data,
+				size: Number(response.headers['content-length']),
+				mimeType: response.headers['content-type'],
+			};
+			const result = await this.uploadFile(userId, params, fileDescription);
+
+			return result;
+		} catch (error) {
+			this.logger.error(`could not find file by url: ${params.url}`, error);
```

Comment:
I do not like logger.error inline. 
The information should pass to the throw error and logged and filtered for outgoing request in the error pipline.